matter/efr32/efr32mg24/BRD4186C/autogen/sl_event_handler.c

```diff
@@ -75,6 +75,9 @@ void sl_service_init(void) {
 void sl_stack_init(void) {
   sl_rail_util_pa_init();
   sl_rail_util_pti_init();
+#if !RS91X_BLE_ENABLE
+  sl_bt_rtos_init();
```

Comment:
same here

Reply:
Hi @mkardous-silabs  reverted the commit